src/core/rate_limiter.h

```diff
@@ -96,6 +96,8 @@ class RateLimiter {
   Status UnregisterModel(const TritonModel* model);
 
   void InitializePayloadQueues(const TritonModelInstance* instance);
+  size_t IdleInstanceCount(const TritonModel* model);
```

Comment:
Where does this function be used?

Reply:
It was needed for one of the variations I tried. But it was not giving best performance. I will clean this up.